packages/headless/src/composables/comboBox/TestCombobox.vue

```diff
@@ -41,16 +41,23 @@ const {
 } = comboBox;
 
 defineExpose({ comboBox });
+
+const handleKeydown = (event: KeyboardEvent) => {
+  switch (event.key) {
+    case "Escape":
+      isExpanded.value = false;
+      break;
+    case "ArrowDown":
+      isExpanded.value = true;
+      break;
+  }
+};
 </script>
 <template>
   <div>
     <label v-bind="label">
       some label:
-      <input
```

Comment:
this a workaround for a bug with vue-tsc: https://github.com/vuejs/language-tools/issues/4369

Reply:
Should we wait for the fix?

Reply:
Sure :)